lib/VL53L0X/VL53L0X.cpp

```diff
@@ -63,7 +63,7 @@ bool VL53L0X::init(bool io_2v8)
   initResult = F(""); // Clear any previous result
   // check model ID register (value specified in datasheet)
   uint8_t modelId = readReg(IDENTIFICATION_MODEL_ID);
-  if (modelId != 0xEE) {
+  if (!(modelId == 0xEE || modelId == 0xEA)) { // Recognize VL53L0X (0xEE) and VL53L1X (0xEA)
```

Comment:
Can't help it, but my brain keeps telling me:
"Why not `(modelId != 0xEE && modelId != 0xEA)` ?"
You don't need to change it, but it took me slightly longer to parse it in my head.

Reply:
You are correct.
I'll probably revert this change though, as the VL53L1X is most likely not properly supported by this plugin/library, and deserves a separate plugin, based on another library. And that other library doesn't support the VL53L0X... 😞 
As the VL53L1X sensor isn't really cheap, I'm not sure I'll order one, as I don't have a purpose for it (yet). I'll think about it some more.